src/core/net_db.cc

```diff
@@ -161,6 +161,11 @@ void NetDb::Run() {
       // builds exploratory tunnels at Nth interval to find more peers
       if (ts - last_exploratory >= static_cast<std::uint16_t>(NetDbInterval::Exploratory)) {
         auto known_routers = GetNumRouters();
+        // TODO(olarks): If I'm not mistaken, this should be initialized with
+        // static_cast<std::uint16_t>(NetDbSize::MinExploratoryTunnels) and then
+        // we can lose the else if block entirely (assuming we truly don't care
+        // about delayed exploratory time (see comment below)). I don't think we
+        // ever should have this as 0 - or else we won't explore.
```

Comment:
Posting for reference to avoid future confusion:

```
&anonimal | olark: but you didn't keep the same logic: the condition was originally *within threshold or if delayed, then explore tunnels*. Now, it's *within threshold, always explore tunnels regardless of delay*.          
&anonimal | olark: take a close look at the placement of the explore calls within the original block, I think you may have gotten confused (based on what I saw on earlier formatting changes).
```